operator/charts/kit-operator/templates/controller/storageclass_io1.yaml

```diff
@@ -0,0 +1,10 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: kit-io1
```

Comment:
Any reason we prefix these with KIT?

Reply:
in my envionment EKS already created a class called gp2. It seems that naming the classes using the storage type is a popular idea. To make sure that we don't have a collision, I give it a prefix "kit-".